examples/src/main/java/tachyon/examples/JournalCrashTest.java

```diff
@@ -0,0 +1,311 @@
+    for (int i = 0; i < sCreateRenameClientNum; i ++) {
+      ClientThread thread = new ClientThread(sTestDir + "/createRename" + i + "/",
+          ClientOpType.CREATE_RENAME_FILE);
+      sClientThreadList.add(thread);
+    }
+
+    // Launch all the client threads and wait for them. If Master crashes, all the threads will
+    // stop at a certain time.
+    for (Thread thread : sClientThreadList) {
+      thread.start();
+    }
+    for (Thread thread : sClientThreadList) {
+      try {
+        thread.join();
+      } catch (InterruptedException e) {
+        LOG.error("Error when waiting thread", e);
+      }
+    }
+
+    // Restart Tachyon Master.
+    String restartMasterCommand = new TachyonConf().get(Constants.TACHYON_HOME)
+        + "/bin/tachyon-start.sh master";
+    try {
+      Runtime.getRuntime().exec(restartMasterCommand).waitFor();
+    } catch (Exception e) {
+      LOG.error("Error when restarting Master", e);
+    }
+    // Wait for Master restart.
+    CommonUtils.sleepMs(null, 1000);
+    boolean isRestart = false;
+    while (!isRestart) {
+      try {
+        // ping Master
+        sTfs.getInfo(new TachyonFile(0));
+        isRestart = true;
+      } catch (Exception e) {
+        // Master has not started.
+      }
+      CommonUtils.sleepMs(null, 1000);
+    }
+
+    // Check status and print pass info.
+    try {
+      if (!checkStatus()) {
+        Utils.printPassInfo(false);
+        System.exit(EXIT_FAILED);
+      }
+      Utils.printPassInfo(true);
+      System.exit(EXIT_SUCCESS);
+    } catch (Exception e) {
+      LOG.error("Failed to check status", e);
+    }
+
+    System.exit(EXIT_FAILED);
+  }
+
+  /**
+   * Parse the input args with a command line format, using
+   * <code>org.apache.commons.cli.CommandLineParser</code>.
+   * @param args the input args
+   * @return true if parse successfully, false otherwise
+   */
+  private static boolean parseInputArgs(String[] args) {
+    Options options = new Options();
+    options.addOption("testDir", true, "Test Directory on Tachyon");
+    options.addOption("creates", true, "create Threads Num");
+    options.addOption("deletes", true, "create/delete Threads Num");
+    options.addOption("renames", true, "create/rename Threads Num");
+    CommandLineParser parser = new BasicParser();
+    CommandLine cmd = null;
+    try {
+      cmd = parser.parse(options, args);
+    } catch (ParseException e) {
+      LOG.error("Failed to parse input args", e);
+      return false;
+    }
+    sTestDir = cmd.getOptionValue("testDir", "/default_tests_files");
+    sCreateFileClientNum = Integer.parseInt(cmd.getOptionValue("creates", "2"));
+    sCreateDeleteClientNum = Integer.parseInt(cmd.getOptionValue("deletes", "2"));
+    sCreateRenameClientNum = Integer.parseInt(cmd.getOptionValue("renames", "2"));
+    return true;
+  }
+
+  private static void printUsage() {
+    System.out.println("Usage: java -cp tachyon-" + Version.VERSION + "-jar-with-dependencies.jar "
```

Comment:
Is there an automatic way for the options to print out the usage, instead of having to do it manually?